h2/src/docsrc/help/help.csv

```diff
@@ -5053,6 +5053,21 @@ Returns the ISO week year from a date/time value.
 ISO_YEAR(CREATED)
 "
 
+"Functions (System)","ABORT_SESSION","
+ABORT_SESSION(sessionInt)
+","
+Cancels the currently executing statement of another session. Closes the session and releases the allocated resources.
+The method only works if the multithreaded kernel is enabled
+(it is enabled by default with MVStore engine only, see SET MULTI_THREADED).
```

Comment:
There is no `SET MULTI_THREADED` command in H2 any more, it should not be mentioned here, take a look on the documentation of `CANCEL_SESSION`:
https://h2database.com/html/functions.html#cancel_session

Reply:
OK, done.